src/pages/product-list.tsx

```diff
@@ -0,0 +1,19 @@
+import Footer from '../components/Footer'
+import Navbar from '../components/Navbar'
+import Sidebar from '../components/Sidebar'
+
+const ProductList = () => (
+  <div className="layout-product-list">
+    <header>
+      <Navbar />
+    </header>
+    <main>
+      <Sidebar />
```

Comment:
Please create a component for product list content since this screen have a sidebar and content.